src/Components/Aspire.RabbitMQ.Client/AspireRabbitMQExtensions.cs

```diff
@@ -172,7 +172,7 @@ private static IConnection CreateConnection(IConnectionFactory factory, int retr
                 if (connectAttemptActivity is not null)
                 {
                     connectAttemptActivity.AddTag("exception.message", ex.Message);
-                    connectAttemptActivity.AddTag("exception.stacktrace", ex.ToString());
+                    connectAttemptActivity.AddTag("exception.stacktrace", ex.StackTrace);
```

Comment:
```suggestion
                    // OTEL's exception.stacktrace attribute includes the type and message.
                    connectAttemptActivity.AddTag("exception.stacktrace", ex.ToString());
```

Reply:
Thanks for catching this. This is a bad name, so adding a comment makes a lot of sense.